lib/Backend/GlobOpt.cpp

```diff
@@ -17435,7 +17435,18 @@ GlobOpt::PreOptPeep(IR::Instr *instr)
         }
         RemoveCodeAfterNoFallthroughInstr(instr);
     }
-
+    if (this->func->HasFinally())
+    {
+        IR::Opnd * dst = instr->GetDst();
+        if (dst && dst->GetSym() && dst->GetSym()->m_id == 0 && instr->m_opcode == Js::OpCode::LdElemI_A)
```

Comment:
Right, it can potentially happen with any opcode with a preop bailout. Corrected this.